DigitalLearningSolutions.Web.Tests/Models/DelegateRegistrationByCentreDataTests.cs

```diff
@@ -25,7 +25,6 @@ public void SetPersonalInformation_sets_data_correctly()
                 LastName = LastName,
                 Centre = CentreId,
                 PrimaryEmail = Email,
-                Alias = Alias

```

Comment:
`private const string Alias = "testuser";` is still declared at the top of the file